e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/helpers/index.ts

```diff
@@ -71,24 +71,27 @@ export const navigateToEntitySelectionStep = (
 
   cy.log("select an experience");
 
-  if (experience === "chart") {
-    cy.findByText("Chart").click();
-  } else if (experience === "exploration") {
-    cy.findByText("Exploration").click();
-  } else if (experience === "browser") {
-    cy.findByText("Browser").click();
+  const hasEntitySelection =
+    experience !== "exploration" && experience !== "metabot";
```

Comment:
is this a little better?


```suggestion
    !["exploration", "metabot"].includes(experience)
```

Reply:
Considering we only have two experiences now `experience !== "exploration" && experience !== "metabot"` is readable to me, but if we need to use this in multiple places then something like (`EXPERIENCES_WITHOUT_ENTITY_SELECTION.includes(experience)`) is better.